platform/ios/src/MGLMapView.h

```diff
@@ -1130,6 +1130,9 @@ MGL_EXPORT IB_DESIGNABLE
 /**
  Converts a geographic bounding box to a rectangle in the given view’s
  coordinate system.
+ 
+ If a longitude is less than −180 degrees or greater than 180 degrees, the
+ bounding box straddles the antimeridian or international date line.
```

Comment:
This method takes a bounding box as input. So maybe adapt the note in `-setVisibleCoordinateBounds:animated:`, which similarly takes a bounding box as input.

(Also in the macOS header.)